### PR TITLE
Add table no

### PR DESCRIPTION
**Pull request by kprakobkit**

### CODE REVIEW

File: test/support/stubs/OrderStub.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import constants from '../../../src/constants';
+
+const OrderStub = {
```
Comment (kprakobkit) [inline test/support/stubs/OrderStub.js:3]: took an OrderStub out into it's own file...this is called a stub, and not a mock, right?

File: src/components/NewOrder.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ import { Link as LinkComponent } from 'react-router';
 
 const MasterItems = createFactory(MasterItemsComponent);
 const Link = createFactory(LinkComponent);
+const tableNumbers = _.range(1, 31);
```
Comment (kprakobkit) [inline src/components/NewOrder.js:10]: Not sure if this belongs here or not. Might be better in the constants file...

Comment (achiu8) [inline src/components/NewOrder.js:10]: Maybe.. doesn't seem like a huge deal for now.

File: test/support/stubs/OrderStub.js (hunk 1 above)
Comment (achiu8) [inline test/support/stubs/OrderStub.js:3]: Yeah, think so.

**Pull request merged**